### PR TITLE
Feat/pagehome

### PR DESCRIPTION
**Pull request by ryanolivrdev**
## Overview
Creation of the home and motel details screens and fix some bugs in the svg, besides fixing the application routes to work with 2 types of navigation

## How to test it
Run `expo start` and navigate the home and click on a card.

## Results
HomePage

Details

## Notes
there is a delay bug in the TabBar when switching from one screen to another

### CODE REVIEW

File: src/components/Button/ButtonRound/index.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { TouchableOpacityProps } from 'react-native';
+import { ButtonContainer } from './styles';
+
+interface ButtonProps extends TouchableOpacityProps {
+  text?: string;
```
Comment (olucasfreitas) [inline src/components/Button/ButtonRound/index.tsx:5]: vapo

File: src/components/Button/ButtonRound/index.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  label: string;
```
Comment (olucasfreitas) [inline src/components/Button/ButtonRound/index.tsx:6]: vapo

File: src/components/Button/ButtonRound/index.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  subtitle: string;
```
Comment (olucasfreitas) [inline src/components/Button/ButtonRound/index.tsx:7]: vapo

File: src/components/Card/CardAccessibility/index.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+  RampWith,
+  TactileFloorOff,
+  TactileFloorWith
+} from '@assets';
+
+import { CardAccessibilityConteiner } from './styles';
+
+interface AccessibilityProps {
+  uneeveness: boolean;
+  elevator?: boolean;
```
Comment (olucasfreitas) [inline src/components/Card/CardAccessibility/index.tsx:22]: conserta

File: src/components/Card/CardAccessibility/index.tsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+  uneeveness,
+  elevator,
+  tactileFloor,
+  signLanguage,
+  braille,
+  bar,
+  incompatibleDimensions
+}: AccessibilityProps) {
+  return (
+    <CardAccessibilityConteiner>
```
Comment (olucasfreitas) [inline src/components/Card/CardAccessibility/index.tsx:40]: CONTEINER  é ruim

File: src/components/Card/CardAccessibilityText/index.tsx [hunk 6]
```diff
@@ -0,0 +1,51 @@
+import {
+  AccessibilityContent,
+  CardAccessibilityTextConteiner,
+  TextAccessibility
+} from './styles';
+
+interface AccessibilityProps {
+  groundfloor: boolean;
+  uneeveness?: boolean;
+  ramp?: boolean;
```
Comment (olucasfreitas) [inline src/components/Card/CardAccessibilityText/index.tsx:10]: conserta

File: src/components/PriceRating/index.tsx [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+interface PriceProps {
+  cost: number;
+}
+
+export function PriceRating({ cost }: PriceProps) {
+  return (
+    <Text accessible={true} style={{ fontWeight: 'bold' }}>
+      <Text
+        style={{
```
Comment (olucasfreitas) [inline src/components/PriceRating/index.tsx:13]: passar pra styled components

File: src/routes/app.routes.tsx [hunk 8]
```diff
@@ -1,14 +1,90 @@
...
+
+  return (
+    <Navigator>
+      <Screen name="Home" component={Home} options={{ headerShown: false }} />
+      <Screen
+        name="MotelDetails"
+        component={MotelDetails}
+        options={{
+          title: '',
+          // tabBarStyle: { display: 'none' },
```
Comment (olucasfreitas) [inline src/routes/app.routes.tsx:47]: vapo

**Review by olucasfreitas — CHANGES_REQUESTED** [review]

**Comment by ryanolivrdev** [comment]
## Overview of the settings
Correct the things discussed and organize the styles of each screen

Home
MotelDetails

**Review by olucasfreitas — APPROVED** [review]

**Pull request merged**